### DISCUSSION TITLE
I updated to the latest version and it no longer works!

### DISCUSSION THREAD

**Discussion by livashu** [PMEAL/OpenPNM#2771] [Q&A]
Input:
import openpnm as op

Output:
Reloaded modules: openpnm.utils._misc, openpnm.utils._settings, openpnm.utils._workspace, openpnm.utils._project, openpnm.utils._health, openpnm.utils, openpnm._skgraph.generators._cubic, openpnm._skgraph.tools._coords_transforms, openpnm._skgraph.tools._funcs, openpnm._skgraph.tools, openpnm._skgraph.generators.tools._funcs, openpnm._skgraph.generators.tools, openpnm._skgraph.operations._funcs, openpnm._skgraph.operations._binary, openpnm._skgraph.operations._unary, openpnm._skgraph.operations, openpnm._skgraph.generators._delaunay, openpnm._skgraph.generators._gabriel, openpnm._skgraph.generators._voronoi, openpnm._skgraph.queries._funcs, openpnm._skgraph.queries._qupc, openpnm._skgraph.queries, openpnm._skgraph.generators._voronoi_delaunay_dual, openpnm._skgraph.generators._template, openpnm._skgraph.generators._fcc, openpnm._skgraph.generators._bcc, openpnm._skgraph.generators, openpnm._skgraph.io._funcs, openpnm._skgraph.io, openpnm._skgraph.simulations._funcs, openpnm._skgraph.simulations._percolation, openpnm._skgraph.simulations, openpnm._skgraph.visualization._funcs, openpnm._skgraph.visualization, openpnm._skgraph, openpnm.core._models, openpnm.core._mixins
Traceback (most recent call last):

  File ~\.spyder-py3\untitled1.py:1 in <module>
    import openpnm as op

  File c:\windows\system32\openpnm\openpnm\__init__.py:13 in <module>
    from . import core

  File c:\windows\system32\openpnm\openpnm\core\__init__.py:50 in <module>
    from ._base2 import *

  File c:\windows\system32\openpnm\openpnm\core\_base2.py:32 in <module>
    @docstr.get_sections(base='BaseSettings', sections=docstr.all_sections)

TypeError: get_sections() missing 1 required positional argument: 's'

Why?

**Comment by jgostick** [comment]
Due to the increasingly high volume of questions that we receive between PoreSpy and OpenPNM, our small team has been finding it very difficult to answer everyone [^1]. For this reason we must decline to do basic "tech-support" for problems which are general python/numpy/programming issues. For these type of questions, you should be able to find the answers you need using a search engine and/or StackOverflow. 

I hope you appreciate that we are not trying to be rude, but just trying to prevent burnout and protect our time to focus on our research activities, including developing new features for PoreSpy and OpenPNM. 

If you find that our documentation is lacking, then we would be very happy to receive an example in the form of a Jupyter Notebook that outlines your solution for future users.  We have an `examples/contrib` folder in both PoreSpy and OpenPNM repos which are intended to house such contributions.  You can either email it to us, or use the pull request functionality of GitHub.

[^1]: This long response was 'auto-generated' using the [saved replies](https://docs.github.com/en/get-started/writing-on-github/working-with-saved-replies/using-saved-replies) feature.
